src/frontend/frontend/console.js

```diff
@@ -283,40 +182,71 @@ angular.module('frontend-app')
     socket.register_callback("test", function(res) {
       self.PIDs = _.without(self.PIDs, res.pid);
       self.PIDs = self.PIDs.length === 0 ? null : self.PIDs;
+      // Try to parse res.stderr for ASAN error output
+      // This string is defined in runner.rkt and should equal it.
+      var asan_marker_string = "this_is_a_special_asan_marker_string_7f84028cdd719df7570532dbf54cd10c3b5a1cc1b4bb62193079da701fdd9acb";
```

Comment:
Please separate out the ASAN error from stderr.  It shouldn't be too hard to do this.